eyes_images/applitools/images/eyes.py

```diff
@@ -73,20 +74,30 @@ def open(self, app_name, test_name, dimension=None):
         # type: (Text, Text, Optional[ViewPort]) -> None
         self.open_base(app_name, test_name, dimension)
 
+    @typing.overload
     def check(self, name, check_settings):
         # type: (Text, ImagesCheckSettings) -> bool
+        pass
+
+    @typing.overload
+    def check(self, check_settings):
+        # type: (ImagesCheckSettings) -> None
+        pass
+
+    def check(self, check_settings_or_name=None, check_settings=None, name=None):
+        check_settings = merge_check_arguments(
```

Comment:
I'm not sure that `merge_check_arguments` should be placed in `eyes_mixin` module

Reply:
Agree

Reply:
I think of putting it to core.eyes_base as a free function. I thought of making it EyesBase staticmethod but selenium.eyes.Eyes is not it's child so calling it will be ugly.

Reply:
Okay, got rid of it totally